src/common/address.h

```diff
@@ -327,9 +336,18 @@ char *tor_dup_ip(uint32_t addr) ATTR_MALLOC;
 MOCK_DECL(int,get_interface_address,(int severity, uint32_t *addr));
 #define interface_address_list_free(lst)\
   interface_address6_list_free(lst)
+/** Free a smartlist of IP addresses returned by get_interface_address_list.
+ */
+static inline void
```

Comment:
Please follow the convention: interface_address_list_free, not free_interface_address_list.

Reply:
Removed in b8645a82ab0bbc7a8adb879024c1ea2e31f5f270.